packages/cli/package.json

```diff
@@ -35,10 +35,10 @@
     "@ast-grep/napi": "0.11.0",
     "@graphql-codegen/cli": "5.0.0",
     "@oclif/core": "2.11.7",
-    "@shopify/cli-kit": "3.50.0",
+    "@shopify/cli-kit": "3.50.2",
     "@shopify/hydrogen-codegen": "^0.1.0",
     "@shopify/mini-oxygen": "^2.2.3",
-    "@shopify/oxygen-cli": "^2.3.2",
+    "@shopify/oxygen-cli": "^2.6.1",
```

Comment:
Should we pin this dependency without the `^`? I think oxygen-cli has automatic bumps and that might mean cli-kit gets out of sync, which is problematic sometimes. For example, the last bump to 3.50 caused a [weird issue](https://github.com/Shopify/hydrogen/pull/1481) in the init flow (that one was an intentional bump, but I guess it could happen without us noticing).